playbooks/ursula-agent.yml

```diff
@@ -0,0 +1,11 @@
+##
+- hosts: all
+  gather_facts: no
+  user: root
+  tasks:
+  - name: download ursula-agent repo
+    git: repo=git@github.blueboxgrid.com:bluebox-tech/ursula-agent.git dest=/tmp/ursula-agent accept_hostkey=yes
+  - name: install deps
+    pip: name=setuptools chdir=/tmp/ursula-agent/ state=latest
+  - name: install ursula-agent
+    shell: /usr/bin/python setup.py install chdir=/tmp/ursula-agent/
```

Comment:
If there is no need for shell redirection, the command module is preferred.

Also, if we're going to use a command like module, we should take advantage of idempotent hints, such as creates= to list a file that the install will create. However, a better method is to make the pip install a handler, so that it only triggers if the git clone results in a change. The git clone task would notify: install ursula-agent   and then we'd move the install ursula-agent task into a handlers: section. That way it's only ran when we're expecting change.

If doing that, /tmp/ursula-agent is probably not the best place, as /tmp can get cleaned. I'd suggest utilizing one of our /opt/ directories, /opt/stack/, or /opt/bbc (not sure which is best, probably a group discussion).